evaluations/tool/src/webapp/content/templates/evaluation_create.html

```diff
@@ -27,6 +27,9 @@
     <!-- end standard on all pages -->
     <link href="../css/evaluation_base.css" type="text/css" rel="stylesheet" media="all" />
     <link rsf:id="local_css_include" href="local_css_file.css" type="text/css" rel="stylesheet" media="all" />
+	<script src="../js/jquery/jquery-1.4.4.min.js" type="text/javascript"></script>
```

Comment:
Is this part of a different change, it doesn't look to be needed to fix the opening in a new window?